x-pack/plugins/security_solution/public/flyout/index.tsx

```diff
@@ -84,42 +79,15 @@ const expandableFlyoutDocumentsPanels: ExpandableFlyoutProps['registeredPanels']
   },
 ];
 
-const OuterProviders: FC = ({ children }) => {
-  return <SecuritySolutionFlyoutUrlSyncProvider>{children}</SecuritySolutionFlyoutUrlSyncProvider>;
-};
-
-const InnerProviders: FC = ({ children }) => {
-  const [flyoutRef, handleFlyoutChangedOrClosed] = useSecurityFlyoutUrlSync();
-
-  return (
-    <ExpandableFlyoutProvider
-      onChanges={handleFlyoutChangedOrClosed}
-      onClosePanels={handleFlyoutChangedOrClosed}
-      ref={flyoutRef}
-    >
-      {children}
-    </ExpandableFlyoutProvider>
-  );
-};
-
+// NOTE: provider below accepts "storage" prop, please take a look into component's JSDoc.
```

Comment:
I'm not sure this comment is doing much. Are you intending to indicate that they may change the component's behavior by changing this hardcoded prop? Why is that important here?